app/Http/Controllers/PrintController.php

```
@@ -91,6 +92,52 @@ public function receipt(Request $request, Sale $sale)
+     * One label per product, each ending in its own cut. F6 of PLAN.md.
+     */
+    public function labels(Request $request)
+    {
+        $user = Auth::user();
+
+        $validated = $request->validate([
+            'product_ids' => ['required', 'array', 'min:1', 'max:100'],
+            'product_ids.*' => ['integer'],
+        ]);
+
+        $query = Product::whereIn('id', $validated['product_ids']);
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Product::whereIn().


(staticMethod.notFound)